src/faz/faz_server.cc

```diff
@@ -44,7 +44,7 @@ Status FazServer::event(ServerContext* context, const EventRequest* request,
     LOG(WARNING) << "event is unhooked for event type " << event_type;
     return Status(StatusCode::NOT_FOUND, "event in not hooked");
   } else {
-    event_function = get_event[0];
+    event_function = get_event[get_event.size() - 1];
```

Comment:
If this is an STL container, it's better to just call `back()` instead of `size()-1`